.readthedocs.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
+
+python:
+   version: 3.7
```

Comment:
Can we use python 3.6 as we have problems with 3.7 on kaolin ? It avoid us to have multiple python versions.

Reply:
Fixed. Python 3.7 issues should be resolved with PR #66, but fixing to python 3.6 in any case as it is the development version and is more regularly tested.